web/turkey/docs/build/html/_static/pygments.css

```diff
@@ -0,0 +1,65 @@
+.highlight .hll { background-color: #ffffcc }
```

Comment:
https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/